application/app/controllers/upload_status_controller.rb

```diff
@@ -1,14 +1,22 @@
 class UploadStatusController < ApplicationController
+  include DetachedProcessStatus
   include LoggingCommon
 
   def index
-    @files = Upload::UploadFilesProvider.new.recent_files
-    ScriptLauncher.new.launch_script
+    files_provider = Upload::UploadFilesProvider.new
+    @files = files_provider.recent_files
+    @summary = FileStatusSummary.compute_summary(@files.map(&:file))
+    uploads = upload_status
+    @status = uploads.idle? ? from_files_summary(@summary) : uploads
```

Comment:
see the comment above regarding the `from_files_summary` bug

Reply:
fixed